docs/best_practices_guide.adoc

```diff
@@ -535,4 +537,79 @@ modules.
 - name: Install etcd (for etcdctl)
   action: "{{ ansible_pkg_mgr }} name=etcd state=latest"
   register: install_result
-  ----
+----
+
+=== Playbook flow and stages
+'''
+[[Playbooks-MUST-use-include_role]]
+[cols="2v,v"]
+|===
+| <<Playbooks-MUST-use-include_role, Rule>>
+| Playbooks MUST use include_role instead of roles:
+|===
+
+[[Playbooks-SHOULD-not-use-the-pre-roles-post-tasks-sections]]
+[cols="2v,v"]
+|===
+| <<Playbooks-SHOULD-not-use-the-pre-roles-post-tasks-sections, Rule>>
+| Playbooks SHOULD not use pre_tasks, post_tasks, and roles: sections.  Should use tasks: and include_role:.
```

Comment:
Consider capitalizing the "not" in "SHOULD not" as without the caps, it isn't being emphasized and while skim reading, I thought that said it "SHOULD" use pre_tasks & co.